nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateCsv.java

```diff
@@ -514,8 +514,9 @@ public void process(OutputStream out) throws IOException {
 
                         } catch (final SuperCsvException e) {
                             valid.set(false);
+                            final String errorMessage = e.getLocalizedMessage() + " " + e.getCsvContext().toString();
```

Comment:
Including the raw csv line could mean printing PII/PID or other sensitive information to logs, or cause very long log outputs when processing large csv files

Should we make the inclusion of the raw csv line optional in this output, what do you think?

I agree that printing the coordinates is a really useful addition